### ISSUE TITLE
[Question] - Crypto usage

### ISSUE THREAD

**Issue by vankk** [brentvollebregt/auto-py-to-exe#97]
Hello, I was trying to use the crypto key and I came across with [this thread](https://stackoverflow.com/a/58077358) on stackoverflow, also tried to use `pip install pycrypto` and didn't worked as well. Seems that the package has [some issues](https://pastebin.com/q3he9uEv), there is any plan to update this feature or just I may doing some wrong?!😛 

*Edit*: nvm, just saw that the pyinstaller still requires the pycrypto and didn't updated yet.

Regards,
vankk.

**Comment by brentvollebregt** [comment]
If you want to use the `--key` argument from PyInstaller, I definitely recommend using `pycryptodome` instead of `PyCrypto` as it is safer. It [has almost the same API as PyCrypto](https://pycryptodome.readthedocs.io/en/latest/src/vs_pycrypto.html) so it should work.

You can install `pycryptodome` using:

```
python -m pip install pycryptodome
```

**Comment by vankk** [comment]
Could you post an usage sample? Did a few searchs the past days and everytime that I open the the .exe returns an `Failed to execute script pyiboot01_bootstrap`.

**Comment by brentvollebregt** [comment]
Outlined in ["Encrypting Python Bytecode"](https://pyinstaller.readthedocs.io/en/stable/usage.html#encrypting-python-bytecode) are some details on how to encrypt your executable contents.

I had been told that `pycryptodome` works with `pyinstaller` but when trying it myself, it appears that may not be the case. I get an error saying: `zlib.error: Error -3 while decompressing data: incorrect header check`.

I have noticed in PyInstallers GitHub issues that the maintainer has mentioned that `--key` is no longer available. I did a search for `--key` in PyInstallers GitHub issues and [found this PR](https://github.com/pyinstaller/pyinstaller/pull/4652) but unfortunately it hasn't been merged yet - looks promising though.
